src/utils/default-plugin.ts

```diff
@@ -1,12 +1,21 @@
-import Module from '../Module';
 import { InputOptions } from '../rollup/types';
+import { Plugin } from '../rollup/types';
 import error from './error';
 import { lstatSync, readdirSync, readFileSync, realpathSync } from './fs'; // eslint-disable-line
 import { basename, dirname, isAbsolute, resolve } from './path';
-import relativeId from './relativeId';
 
-export function load(id: string) {
-	return readFileSync(id, 'utf-8');
+export function getRollupDefaultPlugin(options: InputOptions): Plugin {
+	return {
+		name: 'Rollup Core',
+		resolveId: resolveId(options),
```

Comment:
I wonder if we should rename the rhs function to `getResolveId` or `createResolveId` to distinguish it from the actual hook which is the return value of this function?

Reply:
Would probably trigger some renames in Graph as well, though.